build-image/openhabian.pi-raspios32.conf

```diff
@@ -47,16 +47,16 @@ debugmode=off
 
 # fake hardware
 # force treating your box as if it was a ...
-# pi4, pi4_8gb, cm4, pi400, pi3, cm3, pi3+, cm3+, pi2, pi1, cm1, pi0, pi0w
-#hw=pi4
+# pi4, pi4_8gb, cm4, pi400, pi3, cm3, pi3+, cm3+, pi2, pi1, cm1, pi0, pi0w, x86
+# hw=
 
 # Hardware architecture
 # x86_64, amd64, armv6l, armv7l, aarch64, arm64
-#hwarch=armv7l
+hwarch=armv71
```

Comment:
Do we really want to set by default, why not just leave them as an optional override?

Reply:
I was wondering too. But what for do we have those -raspios32[64].conf files if not to make these differences?

Reply:
The Java differences at build time. We could probably consolidate them, and I can look into it. Not sure if it is necessary.

Reply:
ok now? then pls merge